CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## v2.2.0
+__NOTE:__  This release, fully integrates the new [securesystemslib Signer API
+](https://python-securesystemslib.readthedocs.io/en/latest/signer.html).
+Most notably, runlib API methods receive a new optional argument to sign
+the resulting metadata with a securesystemslib  `Signer` (see securesystemslib
+for available implementations). In addition, the in-toto CLI provides new
+arguments to read consistent standard key file formats for signing (PEM/PKCS8)
+and signature verification (PEM/subjectPublicKeyInfo).
+
+Related legacy key arguments are deprecated and will be removed in the next
+major release. securesystemslib provides a script to [migrate key files
+](https://github.com/secure-systems-lab/securesystemslib#legacy-key-migration).
+
+### Added
+- in-toto-run/record CLI: `--signing-key` arg (#649, #651)
+- in-toto-verify CLI: `--verification-keys` arg (#652)
+- runlib API: `signer` arg (#612)
+- Release automation with PyPI Trusted Publishers (#674)
```

Comment:
Shall we merge #674 and try it out for this release? ping @adityasaky

Reply:
Just approved #674.